resources/config/io.neonbee.internal.verticle.ServerVerticle.example.yaml

```diff
@@ -100,4 +100,23 @@ config:
     - io.neonbee.internal.handler.factories.TimeoutHandlerFactory
     - io.neonbee.internal.handler.factories.SessionHandlerFactory
     - io.neonbee.internal.handler.factories.CacheControlHandlerFactory
-    - io.neonbee.internal.handler.factories.DisallowingFileUploadBodyHandlerFactory
\ No newline at end of file
+    - io.neonbee.internal.handler.factories.CorsHandlerFactory
+    - io.neonbee.internal.handler.factories.DisallowingFileUploadBodyHandlerFactory
+
+    # configures the CORS handler (Disabled by default)
+    cors:
+        enabled: false # Enables the CORS handler.
+        origins: # Set the list of allowed static origins.
+            - http://foo.bar
+        relativeOrigins: # Set the list of allowed relative origins.
+            - "^http\\://.*.foo.bar$"
+        allowedMethods: # Set a set of allowed methods. Upper case required!
+            - GET
+            - PUT
+        allowedHeaders: # Set a set of allowed headers.
```

Comment:
"Set a set" in "Set a set of allowed headers" sounds strange. Why not just something like "The allowed HTTP headers".
Similar case above and below.